CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ Deprecated features will be kept for any following maintenance release, and
 will be removed after two major releases.
 
 ## [Unreleased]
+### Fixed
+- Fix fish completion not escaping characters
```

Comment:
Nit: Might want to expand the changelog given the increased scope!

Reply:
That's a good idea. Looking at the current [CHANGELOG.md](https://github.com/tldr-pages/tldr-c-client/blob/master/CHANGELOG.md), it looks like there is not separate heading for improvements, so I'll just add it under `### Fixed`.

Reply:
Implemented in https://github.com/tldr-pages/tldr-c-client/pull/86/commits/bfd2e99e13b2bb15b2f3a62cea558acff1223fd4